src/bundle/Controller/PasswordResetController.php

```diff
@@ -31,48 +27,37 @@
 use Ibexa\User\View\ResetPassword\FormView as UserResetPasswordFormView;
 use Ibexa\User\View\ResetPassword\InvalidLinkView;
 use Ibexa\User\View\ResetPassword\SuccessView as UserResetPasswordSuccessView;
-use Swift_Mailer;
-use Swift_Message;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
-use Twig\Environment;
 
 class PasswordResetController extends Controller
 {
     private FormFactory $formFactory;
 
     private UserService $userService;
 
-    private Swift_Mailer $mailer;
-
-    private Environment $twig;
-
     private ActionResultHandler $actionResultHandler;
 
     private PermissionResolver $permissionResolver;
 
     private ConfigResolverInterface $configResolver;
 
-    private NotificationServiceInterface $notificationService;
+    private Notifier $passwordResetMailer;
```

Comment:
```suggestion
    private PasswordResetNotifierInterface $passwordResetMailer;
```

Reply:
Class name is `Ibexa\Contracts\User\PasswordReset\Notifier`; you want me to import this with alias or you just missread implementation and interface names?

Reply:
There is/was no such convention - we were doing it only when interface was in same namespace as implementation.

Reply:
I agree with Konrad. We have had discussions about it, but without a strong conclusion. For me, this case is an example of when it is easier to read and understand what's going on when we have an Interface suffix and a more meaningful name than 'Notifier'